pkg/operator/ceph/provisioner/provisioner_test.go

```diff
@@ -76,7 +76,7 @@ func TestProvisionImage(t *testing.T) {
 	assert.Equal(t, "pvc-uid-1-1", pv.Name)
 	assert.NotNil(t, pv.Spec.PersistentVolumeSource.FlexVolume)
 	assert.Equal(t, v1.PersistentVolumeReclaimRetain, pv.Spec.PersistentVolumeReclaimPolicy)
-	assert.Equal(t, "foo.io/rook-system", pv.Spec.PersistentVolumeSource.FlexVolume.Driver)
```

Comment:
why did these asserts change?  i'm not sure i see the reason yet

Reply:
This was a subtle change... The mocking changed such that the version of k8s is coming back as `master` instead of `1.11.3` or some other specific version. The side effect is that the checks for k8s version in the `RookDriverName` function doesn't know what version it's running, so it falls back to return the default flex driver name `rook`.

Reply:
The flex volume directory detection now is based only on the rook env variable or the default location. There is also a unit test added `TestDiscoverFlexDir`. It's already squashed to get the build going and potentially merged sooner.